gnome-extra/krb5-auth-dialog/krb5-auth-dialog-43.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson gnome2-utils
+
+DESCRIPTION="An application to acquire and list Kerberos tickets."
+HOMEPAGE="https://gitlab.gnome.org/GNOME/krb5-auth-dialog/"
+SRC_URI="https://download.gnome.org/sources/${PN}/$(ver_cut 1)/${P}.tar.xz"
+
+LICENSE="GPL-2"
```

Comment:
I looked at a few source files, it seems this should be `GPL-2+`.  Also the documentation is `FDL-1.1+`.  Please check and update accordingly

Reply:
i did diff /usr/portage/licenses/GPL-2 against the COPYING file in the sources, diff is whitespace only so i will assume it is GPL-2

for anything else we could likely get back to upstream at some point, but for the moment i guess i will keep trying with "GPL-2"

Reply:
Please read the relevant pages from the devmanual: 
- [Determining the correct license](https://devmanual.gentoo.org/general-concepts/licenses/#determining-the-correct-license)
- [GPL-x vs GPL-x+](https://devmanual.gentoo.org/general-concepts/licenses/#gpl-x-vs-gpl-x)

Reply:
Let us see what upstream has to say about that

https://gitlab.gnome.org/GNOME/krb5-auth-dialog/-/issues/12

Reply:
In the meantime i would just take whichever value you suggest @ceamac , please provide the exact line so i can just drop it in.

Reply:
i took `LICENSE="GPL-2+ FDL-1.1+`